README.md

```diff
@@ -21,6 +21,19 @@ To bind the java agent to a specific IP change the port number to `host:port`.
 
 See `./run_sample_httpserver.sh` for a sample script that runs the httpserver against itself.
 
+### Wildfly
```

Comment:
Would you prefer me to move the docs to the wildfly example configuration or a separate .md file in the repo?